src/index.html

```diff
@@ -8,6 +8,76 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <h1>Stars</h1> 
+    
+  </svg>
+    <div class="container">
+      <ul class="stars stars--0">
+        <li class="stars__star"></li>
+    
+        <li class="stars__star"></li>
+    
+        <li class="stars__star"></li>
+    
+        <li class="stars__star"></li>
+    
+        <li class="stars__star"></li>
+      </ul>
+      <ul class="stars stars--1">
```

Comment:
```suggestion
    <div class="container">
      <ul class="stars stars--0">
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
      </ul>
      
      <ul class="stars stars--1">
```
Better add empty lines between different blocks